src/pgrepr-consts/src/oid.rs

```diff
@@ -454,6 +454,7 @@ pub const TABLE_MZ_CLUSTER_REPLICAS_OID: u32 = 16728;
 pub const TABLE_MZ_INTERNAL_CLUSTER_REPLICAS_OID: u32 = 16729;
 pub const TABLE_MZ_CLUSTER_REPLICA_STATUSES_OID: u32 = 16730;
 pub const TABLE_MZ_CLUSTER_REPLICA_SIZES_OID: u32 = 16731;
+pub const TABLE_MZ_UNSTABLE_CLUSTER_REPLICAS_OID: u32 = 16732;
```

Comment:
Can you add this to the end of the file and use max+1 instead of skipped oid in the middle?